### DISCUSSION TITLE
Is there a way of exporting SETTINGS (autocorrect values in particular)?

### DISCUSSION THREAD

**Discussion by dajare** [Zettlr/Zettlr#2870] [Q&A] [answered]
I've read the docs, and explored the interface but cannot see any way of exporting SETTINGS that could be then imported in another Zettlr.

In particular, I have a longish set of "autocorrect" options that is a pain to enter manually. If I could *somehow* transfer these to a Zettlr instance on another machine, that would be wonderful.

FWIW, I've got v. 2.0.3 running on MacOS (still on Mojave) at home, but work machine is running Zettlr on Ubuntu.

Is there any hope? or do I have to type these all manually again? Thanks!

**Comment by nathanlesage** [comment, marked as answer]
The settings are written as JSON in your App Data directory (i.e. `/Users/username/Library/Application Support/Zettlr`), called `config.json`. Everything's in there. I was pondering to have Zettlr point to a specific configuration directory so that (platform-independent) settings can be shared across apps using a cloud provider, but since there are a few platform-dependant settings in there I first have to separate those settings from each other, and then …

But you should be mostly fine to display the settings and copy out anything of use. There are some online-editors that should let you have a look at the JSON to see you're copying the values correctly (especially since Zettlr minifies the JSON)

**Reply by dajare** [reply]
Thanks for the reply - sounds quite promising! I'll take a look at that this evening when I'm back on MacOS machine (now on Ubuntu :wink: ).

**Reply by dajare** [reply]
And just to update: I've now done that, and it worked perfectly. :+1:  Thanks for this!

**Reply by nathanlesage** [reply]
Wonderful!
